tests/problem2.test.js

```diff
@@ -22,48 +22,63 @@
  * przyjmować po prostu listy Promise'ów?
  */
 
-const p = require('../utils.js');
+const p = require("../utils.js");
 
-describe('problem2', () => {
-    it('properly resolves with just one promise', async () => {
-        const result = await attempt([
-            () => p.resolveWith(1).after(10)
-        ]);
+const attempt = functions => {
+  return new Promise(async (resolve, reject) => {
```

Comment:
podobnie jak w poprzednim zadaniu, to opakowywanie `async` w `new Promise` wygląda bardzo dziwnie.

Da się w tym kodzie usunąć `new Promise` zupełnie i tak zmienić funkcję w środku, żeby wszystko działało. :)